src/scss/includes/component.scss

```diff
@@ -2,6 +2,7 @@
 
 $panel_width : 400px;
 $top_bar_height: 64px;
+$ui_margin: 12px;
```

Comment:
Shouldn't this be `u-m-12` in `utils.scss` ?

Not necessarily in the context of this PR, but I think we should define spaces of the design system.

Reply:
This is different, for two reasons:
 - this is an individual value, not a class like `.u-mb-12` and such ;
 - more importantly, it has a semantic (it's the main external margin of the whole UI), it's not just some generic util.

But I agree we should have a discussion about vars and utils in our own small version of the design system. For example, I noticed the Zeplin designs use named size values, like `-xl`, `-xs`, etc. Maybe we should move to that eventually. We could then have these as vars, and utils using them. For example:
```scss
$size-s: 8px;
$size-l: 16px;
// ...
.u-mb-s {
   margin-bottom: $size-s;
}
.u-mb-l {
   margin-bottom: $size-l;
}
.u-mr-s {
   margin-right: $size-s;
}
.u-mr-l {
   margin-right: $size-l;
}
// ...
```
Or just see how it's done in the "real" DS and just copy/paste it.
Let's organize something to talk about it:)